bridge_ui/src/config.ts

```diff
@@ -0,0 +1,33 @@
+import { ChainId, CHAIN_ID_POLYGON } from "@certusone/wormhole-sdk";
+
+export type DisableTransfers = boolean | "to" | "from";
+
+export interface WarningMessage {
+  text: string;
+  link?: {
+    url: string;
+    text: string;
+  };
+}
+
+export interface ChainConfig {
+  disableTransfers?: DisableTransfers;
+  warningMessage?: WarningMessage;
+}
+
+export type ChainConfigMap = {
+  [key in ChainId]?: ChainConfig;
+};
+
+export const CHAIN_CONFIG_MAP: ChainConfigMap = {};
+
+CHAIN_CONFIG_MAP[CHAIN_ID_POLYGON] = {
```

Comment:
type checking doesn't work on this object's properties when it's included in the initialization of CHAIN_CONFIG_MAP...

Reply:
It will if you use `as ChainConfigMap` after :wink:

Reply:
Changed